skins/oasis/modules/NotificationsController.class.php

```diff
@@ -7,14 +7,6 @@
 
 class NotificationsController extends WikiaController {
 
-	const SESSION_KEY = 'oasisConfirmation';
```

Comment:
(Not related to this line, but) should the doc block on line 3 be updated to remove the reference to "(green stripe below global nav)"? It looks like you split this file out into the two types of notifications (which is good).